tests/Integration/Behaviour/Features/Context/Domain/ProductFeatureContext.php

```diff
@@ -229,6 +261,48 @@ public function assertLastErrorIsInvalidTypeConstraint()
         );
     }
 
+    /**
+     * @Then I should get error that product description is invalid
+     */
+    public function assertLastErrorIsInvalidDescriptionConstraint()
+    {
+        $this->assertLastErrorIs(
+            ProductConstraintException::class,
+            ProductConstraintException::INVALID_DESCRIPTION
+        );
+    }
+
+    /**
+     * @Then I should get error that product short description is invalid
+     */
+    public function assertLastErrorIsInvalidShortDescriptionConstraint()
+    {
+        $this->assertLastErrorIs(
+            ProductConstraintException::class,
+            ProductConstraintException::INVALID_SHORT_DESCRIPTION
+        );
+    }
+
+    /**
+     * Map a technical property name with a more user oriented name from scenario.
+     *
+     * @param string $fieldName
+     *
+     * @return string
+     */
+    private function getPropertyByFieldName(string $fieldName): string
```

Comment:
This seems overly complex to me, it would be easier to simply use `description_short` in the sentence I get the human readable idea but I'm not sure it has such a good complexity/value ratio

Reply:
it might have more value on other fields (like if its integer but needs to reflect some word), but ok, lets stick with simple solution while we can.